### PR TITLE
added unnormalization in wrapper

### PR DESCRIPTION
**Pull request by VictorGillioz**
This PR add a method in the wrapper to unnormalize predictions when doing a forward pass. It also modifies the visualization file to follow this new implementation and offer the option to plot both normalized and unnormalized

### CODE REVIEW

File: meshnets/modules/lightning_wrapper.py [hunk 1]
```diff
@@ -45,9 +45,22 @@ def normalize_labels(self, y: torch.Tensor) -> torch.Tensor:
...
         return y
 
+    @torch.no_grad()
+    def unnormalize_labels(self, y: torch.Tensor) -> torch.Tensor:
+        """Unnormalize the labels in a batch.
+        
+        The labels are unnormalized according to the mean and std given
+        to the wrapper."""
+
+        # Send the stats to the same device as the data and normalize
```
Comment (AugustoPeres) [inline meshnets/modules/lightning_wrapper.py:55]: This comment is unnecessary

**Review by AugustoPeres — APPROVED** [review]
LGTM
